apigentools/utils.py

```diff
@@ -414,3 +414,28 @@ def validate_duplicates(loaded_keys, full_spec_keys):
     for key in loaded_keys:
         if key in full_spec_keys:
             raise ValueError("Duplicate field {} found in spec. Exiting".format(key))
+
+
+def volumes_from(alt_volumes):
+    retval = []
+    is_image_run = env_or_val("APIGENTOOLS_IMAGE", None)
+    if is_image_run:
+        if os.path.exists("/proc/self/cgroup"):
+            print("exists")
+            with open("/proc/self/cgroup") as f:
+                for line in f.readlines():
+                    print(line)
```

Comment:
Are those two debug logs ?

Reply:
Yeah, forgot to remove these... good catch :)